scripts/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export REGISTRY=quay.io/kubernetes-service-catalog
```

Comment:
@duglin I haven't created that quay organization yet because I'm waiting for an answer on https://github.com/kubernetes-incubator/service-catalog/pull/529#discussion_r105745840. 

Regardless, I agree that we don't need image prefixes either way, which means we can probably close https://github.com/kubernetes-incubator/service-catalog/issues/331 with this PR.